app/assets/stylesheets/template/total_amount.css

```diff
@@ -0,0 +1,68 @@
+.btn-widget {
```

Comment:
**Comment on the file:** 
1. file location shouldn't be in template: only template stylesheets are kept their.
2. please use sass extension `.scss` in all new stylesheet files